environment/src/main/java/com/sequenceiq/environment/network/service/extended/MlxSubnetConstants.java

```diff
@@ -0,0 +1,14 @@
+package com.sequenceiq.environment.network.service.extended;
+
+public class MlxSubnetConstants {
+
+    public static final String MLX_SUBNET_MASK = "24";
+
+    public static final int MLX_SUBNET_STARTING_IP = 3;
+
+    public static final int MLX_SUBNET_FINAL_IP = 34;
```

Comment:
I think this should be the count of subnets

Reply:
I originally implemented this way because we needed to know the starting and final IPs of the different ranges in order to determine the subnet types, without storing it in the database. Since it is stored now in the database, it can be changed to count.

Reply:
done